### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by sim0nx**

### CODE REVIEW

File: rt/rest2.py [hunk 1]
```diff
@@ -748,6 +754,26 @@ def reply(self,
...
+        """ Mark a ticket as deleted.
+
+        :param ticket_id: ID of ticket
+
+        :raises BadRequestError: When user does not exist
+        :raises NotFoundError: If the user does not exist
+        :raises UnexpectedResponseError: If the response from RT is not as expected
+        """
+        try:
+            _ = self.__request_delete(f'ticket/{ticket_id}')
```
Comment (sebix) [inline rt/rest2.py:767]: Why storing the return value if it's not used?

File: rt/rest2.py [hunk 2]
```diff
@@ -649,6 +651,10 @@ def edit_ticket(self, ticket_id: typing.Union[str, int], **kwargs: typing.Any) -
         if not msg:
             return True
 
+        for k in msg:
+            if re.search(REGEX_PATTERNS['does_not_exist'], k):
```
Comment (sebix) [inline rt/rest2.py:655]:
```suggestion
            if REGEX_PATTERNS['does_not_exist'].search(k):
```

File: rt/rest2.py (hunk 1 above)
Comment (sim0nx) [reply rt/rest2.py]: absolutely, fixed

File: rt/rest2.py (hunk 2 above)
Comment (sim0nx) [reply rt/rest2.py]: :facepalm: ... thnx for catching that :+1:

Comment (sim0nx) [reply rt/rest2.py]: Sure that's also why I didn't notice it when testing, but it should not be called like that, looks ugly :-)

**Pull request merged**